pyccel/ast/basic.py

```diff
@@ -1,3 +1,6 @@
+#This file is part of pyccel which is released under MIT License.
+#See file LICENSE or go to https://github.com/pyccel/pyccel/blob/master/LICENSE
+#for full license details.
```

Comment:
Can we put an empty space after the `#` symbol? This is customary for every comment in Python.

Reply:
I also think that we should separate these 3 lines a bit from the other comments. Either with an empty line before/after them, or with a explicit separator. E.g.
```python
#--------------------------------------------------------------------------------------#
# This file is part of Pyccel which is released under MIT License. See file LICENSE or #
# go to https://github.com/pyccel/pyccel/blob/master/LICENSE for full license details. #
#--------------------------------------------------------------------------------------#
```
@saidctb @EmilyBourne @ratnania What do you think?
```

Reply:
Looks good to me

Reply:
it looks good to me too